src/commands/serve.js

```diff
@@ -58,6 +58,14 @@ module.exports = new Command("serve")
   })
   .action((options) => {
     options.targets = filterOnly(VALID_EMULATORS, options.only);
+    if (options.targets.indexOf("database") >= 0 || options.targets.indexOf("firestore") >= 0) {
+      utils.logWarning(
+        `${clc.bold(
+          "firebase emulators:start"
+        )} is the preferred way to start the Realtime Database or Cloud Firestore emulators.`
```

Comment:
Please the wording actionable, like: "Please use `firebase emulators:start` instead to start ..."

Also question: are we deprecating / removing `serve --only`?

Reply:
In 8.0 we will almost definitely remove `database` and `firestore` from `serve` and make it a Hosting/Functions thing only (and even for Functions we will still warn people it's not the best way)